dataset_conversion/nnunetV2_to_bids.py

```diff
@@ -0,0 +1,106 @@
+"""
+Converts nnUNetv2 dataset format to the BIDS-structured dataset. Full details about
+the format can be found here: https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/dataset_format.md
+
+Théo Mathieu
+"""
+
+import argparse
+import numpy as np
+import shutil
+import pathlib
+from pathlib import Path
+import json
+import os
+import nibabel as nib
+
+def get_parser():
+    # parse command line arguments
+    parser = argparse.ArgumentParser(description='Convert BIDS-structured dataset to nnUNetV2 database format.')
+    parser.add_argument('--path-bids', required=True, help='Path to nnUNet dataset. Example: ~/data/dataset')
+    parser.add_argument('--path-out', required=True, help='Path to output directory. Example: ~/data/dataset-bids')
+    parser.add_argument('--suffix', required=True, help='Suffix of the label file Example: sub-003_T2w_SUFFIX.nii.gz')
+    parser.add_argument('--copy', '-cp', type=bool, default=False,
+                        help='Making symlink (False) or copying (True) the files in the Bids dataset. '
+                             'This option only affects the image file, the label file is copied regardless of the '
+                             ' option, default = False. Example for symlink: ø, Example for copy: --copy')
+    return parser
+
+
+
+def get_subject_info(file_name, contrast_dict):
+    """
+    Get different information about the current subject
+
+    Args:
+        file_name (str): Filename corresponding to the subject image
```

Comment:
Could you add an example file_name here?

Reply:
Done in https://github.com/ivadomed/utilities/pull/15/commits/5d0b9fcbb66554eb32982e42582eef28fa676e6b